Resources/doc/cookbook/registration_form.rst

```diff
@@ -254,7 +254,7 @@ the validation and saves the data into MongoDB:
 
     public function createAction(Request $request)
     {
-        $dm = $this->get('doctrine_mongodb')->getManager();
+        $dm = $this->get(DocumentManager::class);
```

Comment:
I wasn't fixing `$this->get` calls just yet, but I can do it now, just need to figure out how to show that we're injecting DocumentManager. Or should we do method injection?

Reply:
I'm all for using method injection: it's enabled by default and it avoids having to add a constructor for DI purposes to the code samples.

Reply:
Done 👍